articles/140_topic_and_service_name_mapping.md

```diff
@@ -243,41 +254,76 @@ Here is a non-exhaustive list of prefixes:
 | ROS Parameter        | rp     |
 | ROS Action           | ra     |
 
-While all planned prefixes consist of two characters, i.e. `rX`, anything proceeding the first namespace separator, i.e. `__`, can be considered part of the prefix.
+While all planned prefixes consist of two characters, i.e. `rX`, anything proceeding the first namespace separator, i.e. `/`, can be considered part of the prefix.
 The standard reserves the right to use up to 8 characters for the prefix in case additional prefix space is needed in the future.
 
+### Examples of ROS Names to DDS Concepts
+
+Here are some examples of how a fully qualified ROS name would be broken down into DDS concepts:
+
+| ROS Name                        | DDS Topic   | DDS Partition               |
+|---------------------------------|-------------|-----------------------------|
+| `/foo`                          | `foo`       | [`/rt/`]                    |
+| `rostopic:///foo/bar`           | `bar`       | [`/rt/foo/`]                |
+| `/robot1/camera_left/image_raw` | `image_raw` | [`/rt/robot1/camera_left/`] |
+
+### Changing ROS Names During Runtime
+
+If the ROS name needs to be changed during runtime (after the publisher or subscriber has already been created and is in use), then some details about DDS partitions might be useful to consider.
+
+Since this proposal splits the ROS name into namespace and base name, we'll consider how that might affect DDS partitions and DDS topic names respectively.
+When changing the namespace of a ROS name, i.e. everything but the base name, then this could be done by changing just the existing partition string.
+As partitions are part of QoS settings, it technically doesn't require a full restart of the publisher and subscription when changing the partitions field, however we cannot change the DDS topic name on the fly.
+With this in mind, we may want to always require a full restart of the publisher or subscription instance to avoid inconsistency in how these changes behave from the ROS user's perspective.
```

Comment:
Yeah, I left this note, or implementation hint, in because @Karsten1987 went to the trouble to test it out and I didn't want to lose the information. But I agree that the API should make it clear that you should just assume that the publisher or subscriber may need to be recreated, i.e. we shouldn't try to make any guarantee about the possible shortcut that can be taken when the basename is unchanged.